tests/visual/slider.html

```diff
@@ -605,6 +605,54 @@
                 </div>
             </div>
         </section>
+
+        <span class="k-rtl">RTL Horizontal (Angular)</span>
+        <span></span>
+        <span></span>
+
+        <section class="k-rtl">
+            <kendo-slider class="k-slider-horizontal k-slider k-widget k-state-default k-slider-transitions" style="width: 276px;">
```

Comment:
Could you please add `dir="rtl"` to the `<kendo-slider> element and remove the `k-rtl` class from the `<section>`?
That's how it works in Kendo Angular - the `dir` attribute is rendered on the component host element.